integration_test/deployment-cumulus-orca.sh

```diff
@@ -70,7 +71,6 @@ terraform apply \
   -var "urs_client_password=$bamboo_EARTHDATA_CLIENT_PASSWORD" \
   -var "urs_url=https://uat.urs.earthdata.nasa.gov" \
   -var "api_users=[\"bhazuka\", \"andrew.dorn\", \"rizbi.hassan\", \"scott.saxon\"]" \
-  -var "cmr_oauth_provider=$bamboo_CMR_OAUTH_PROVIDER" \
```

Comment:
No value for the variable. Was always blank.

Reply:
this should stay according to https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse/cumulus-tf/terraform.tfvars.example?at=refs%2Fheads%2Frelease%2Fv11.1.1-v4.0.1#80

Reply:
I don't follow. I see that the example tfvars sets this value, but that doesn't translate into requiring this override.

What this line was actually doing was overwriting the default "earthdata" with "" since the variable `$bamboo_CMR_OAUTH_PROVIDER` was never exposed/set.

Reply:
from what I know, cmr_oauth_provider should either be set as "earthdata" or "launchpad" that's why I kept that as a variable that the user can replace if needed.

Reply:
It was never set as a variable in bamboo.yaml, so from my understanding, the user could not replace it.

Additionally, it was never given a value of "earthdata" or "launchpad", so what this line was actually doing was overwriting the default "earthdata" with "".

Reply:
I would add this variable in spec file
`CMR_OAUTH_PROVIDER: earthdata`

Reply:
Added back, and added default.